i18n/en.pot

```diff
@@ -492,6 +492,9 @@ msgstr ""
 msgid "External authentication only (OpenID or LDAP)"
 msgstr ""
 
+msgid "Expire date"
```

Comment:
Not sure if this should perhaps be `Expiry date` / `Expiration date`.
Perhaps @cooper-joe could sign off?

Reply:
I would go for `Expiration date` 👍

Reply:
The field on the user credentials is called `accountExpiry`, so maybe `Account expiration date`?

Reply:
Yes, I think `Account expiration date` makes a lot of sense in that case